src/common/box_filters.c

```diff
@@ -385,7 +385,7 @@ static void blur_vertical_4ch_sse(float *const restrict buf, const size_t height
   }
   // process the blur up to the point where we start removing values
   size_t y;
-  for (y = 0; y <= MIN(radius, height-1); y++)
+  for (y = 0; y <= MIN(radius, height-radius-1); y++)
```

Comment:
Can you explain and double check this one ? This is quite a big difference and does not correspond really to the description of the commit. Is that intended ? A typo while editing ? TIA.

Reply:
This prevents a buffer overrun when 2*radius > height.  There are some other occurrences in the code, but this one was missed previously.  I believe that this change is what fixes #8066.

Reply:
Understood, thanks.